x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/mapper/DoubleScriptFieldTypeTests.java

```diff
@@ -183,7 +183,9 @@ public void testRangeQuery() throws IOException {
 
     @Override
     protected Query randomRangeQuery(MappedFieldType ft, SearchExecutionContext ctx) {
-        return ft.rangeQuery(randomLong(), randomLong(), randomBoolean(), randomBoolean(), null, null, null, ctx);
+        long lower = randomLongBetween(0, 1000);
+        long upper = lower + randomLongBetween(5, 5000);
+        return ft.rangeQuery(lower, upper, randomBoolean(), randomBoolean(), null, null, null, ctx);
     }
```

Comment:
Because we're actually executing the random query now, we need to ensure its parameter constraints are met.